_data/toc.yaml

```diff
@@ -1196,8 +1196,6 @@ manuals:
           title: Logs and troubleshooting
         - path: /desktop/mac/apple-silicon/
           title: Apple silicon
-        - path: /desktop/mac/release-notes/
-          title: Release notes
```

Comment:
@usha-mandya I removed these from the toc, but haven't removed the actual files yet. I figured we'd want to keep them for a little bit, just in case? Was this correct?